webapp/channels/src/components/admin_console/schema_admin_settings.tsx

```diff
@@ -1108,6 +1108,23 @@ export class SchemaAdminSettings extends React.PureComponent<Props, State> {
                     );
                 }
 
+                // This is a bit of special case since designs for plugin config expect the Enable/Disable setting
+                // to be on top and out of the sections.
+                if (section.key.startsWith('PluginSettings.PluginStates') && section.key.endsWith('Enable.Section')) {
```

Comment:
I'm just brainstorming with this, but I wonder if it would make sense to genericize this by exposing a `section.variant` or something similar to let arbitrary sections render with `container={false}` and then make the enable/disable setting pass that

Reply:
Yeah, that's possibly another option. I just found it a bit of a weird case as I'd expect at least a section container while here we are using a settings group so we are mixing things (settings and sections) but that comes from a design requirement.